compiler/rustc_passes/src/check_attr.rs

```diff
@@ -717,6 +717,42 @@ impl CheckAttrVisitor<'tcx> {
         true
     }
 
+    /// Checks that `doc(test(...)` attribute contains only valid attributes. Returns `true` if
+    /// valid.
+    fn check_test_attr(&self, meta: &NestedMetaItem, hir_id: HirId) -> bool {
+        let mut is_valid = true;
+        if let Some(metas) = meta.meta_item_list() {
+            for i_meta in metas {
```

Comment:
Can you test how this behaves for `doc(test())`? I'd expect it to do nothing.

Reply:
Sure.